fxa-auth-db-server/index.js

```diff
@@ -94,6 +94,11 @@ function createServer(db) {
       return db.getSecondaryEmail(Buffer(req.params.email, 'hex'))
     })
   )
+  api.get('/account/email/:email',
```

Comment:
Without wanting to bikeshed the whole API here, I wonder if `/email/:email/account` would fit better with the pattern of URL structures used here.  Just a suggestion, feel free to take it or leave it :-)